aws/ec2/ec2/vm.go

```diff
@@ -12,11 +12,11 @@ import (
 )
 
 type VM struct {
-	context    *pulumi.Context
-	runner     *command.Runner
-	aptManager *command.AptManager
+	context *pulumi.Context
+	runner  *command.Runner
 
 	CommonEnvironment *config.CommonEnvironment
+	PackageManager    *command.AptManager
```

Comment:
Should be the `interface` type

Reply:
Just package manager or the VM ? I thought we agreed on using a struct as we only have one implementation of VM so far

https://github.com/DataDog/test-infra-definitions/pull/10#discussion_r1035826425

Reply:
The `PackageManager`, the `command.PackageManager` interface already exists.

Reply:
I understand now, it should be `*command.PackageManager` type, which is an `interface`